.eslintrc.js

```diff
@@ -15,6 +18,29 @@ module.exports = {
     defineModel: 'readonly'
   },
   rules: {
+    '@intlify/vue-i18n/no-unused-keys': [ 'error', {
+      src: 'src/',
+      callExpression: '^(\\$t|t|\\$tc|tc|\\$tcn|tcn|\\$tn|tn)$',
```

Comment:
Re: being blocked by intlify/eslint-plugin-vue-i18n#643, I have two thoughts:

1. Maybe we just ignore `$tcn()` and `tn()` for now. They're not the most common way to access i18n messages.
2. What if in CI, we modified code files, e.g., replacing all strings containing `/\btn\(/` with `t(`? Would the Vue I18n check work then?